app/controllers/admin/divisions_controller.rb

```diff
@@ -11,11 +12,101 @@ def select
     redirect_to redisplay_url
   end
 
+  def index
+    authorize Division
+    @divisions_grid = initialize_grid(
+      policy_scope(Division),
+      conditions: index_filter,
+      order: 'name',
+      per_page: 50
+    )
+    @parent_filter_choices = current_user.accessible_divisions.map{ |d| [d.name, d.id] }
+  end
```

Comment:
Please add the csv export boilerplate which is now in e.g. loans_controller and loans/index view. It's just a few lines here and a few in the template (grep for csv in the template to find the special stuff there).